docs/docs-content/clusters/edge/edgeforge-workflow/prepare-user-data.md

```diff
@@ -272,6 +272,36 @@ stages:
           passwd: kairos
 ```
 
+### Create Bind Mounts
+
+Palette Edge allows you to create bind mounts from your Edge host to your cluster through the installer configuration
+file named **user-data**, which allows your cluster to use directories or files from your Edge host directly within your
+Kubernetes cluster. This setup is useful for scenarios where your applications running in the cluster need direct access
+to files or directories on the Edge host.
+
+Several packs require you set up bind mounts in order to function. For example, the Calico pack requires the folder
```

Comment:
Instead of using Calico here, could we do the ones that don't have an example, such as tailscale and Contained? Portworx we can also add to the its pack's page